setup.py

```diff
@@ -5,12 +5,12 @@
 
 
 def read(fname):
-    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+    return open(os.path.join(os.path.dirname(__file__), fname), encoding="utf-8").read()
```

Comment:
caused some errors on Windows in some cases, if the encoding couldn't be inferred properly.